spec/support/schema_matchers.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require 'rspec/expectations'
+
+module SchemaMatchers
```

Comment:
What's the rationale behind making this a module that needs to be mixed in?  Defining `Rspec::Matchers` inline within the file seemed ok since all specs get it from the `rails_helper`/`spec_helper` combo.